[2023-06-10T01:51:00.021+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: fhv_taxi_data.format_to_parquet_task scheduled__2020-08-02T07:00:00+00:00 [queued]>
[2023-06-10T01:51:00.044+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: fhv_taxi_data.format_to_parquet_task scheduled__2020-08-02T07:00:00+00:00 [queued]>
[2023-06-10T01:51:00.045+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2023-06-10T01:51:00.045+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 2
[2023-06-10T01:51:00.045+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2023-06-10T01:51:00.065+0000] {taskinstance.py:1383} INFO - Executing <Task(PythonOperator): format_to_parquet_task> on 2020-08-02 07:00:00+00:00
[2023-06-10T01:51:00.074+0000] {standard_task_runner.py:54} INFO - Started process 2925 to run task
[2023-06-10T01:51:00.087+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'fhv_taxi_data', 'format_to_parquet_task', 'scheduled__2020-08-02T07:00:00+00:00', '--job-id', '145', '--raw', '--subdir', 'DAGS_FOLDER/data_ingestion_gcs_multiFile_v02.py', '--cfg-path', '/tmp/tmpwrkq1kgu']
[2023-06-10T01:51:00.088+0000] {standard_task_runner.py:83} INFO - Job 145: Subtask format_to_parquet_task
[2023-06-10T01:51:00.089+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/***/dags/data_ingestion_gcs_multiFile_v02.py
[2023-06-10T01:51:00.207+0000] {data_ingestion_gcs_multiFile_v02.py:20} INFO - dtc_data_lake_dtc-dezc-001 is the bucket.
[2023-06-10T01:51:00.230+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/configuration.py:545: DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
  option = self._get_environment_variables(deprecated_key, deprecated_section, key, section)

[2023-06-10T01:51:00.385+0000] {task_command.py:384} INFO - Running <TaskInstance: fhv_taxi_data.format_to_parquet_task scheduled__2020-08-02T07:00:00+00:00 [running]> on host f41d61956f54
[2023-06-10T01:51:00.479+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:204: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2023-06-10T01:51:00.607+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=admin
AIRFLOW_CTX_DAG_ID=fhv_taxi_data
AIRFLOW_CTX_TASK_ID=format_to_parquet_task
AIRFLOW_CTX_EXECUTION_DATE=2020-08-02T07:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2020-08-02T07:00:00+00:00
[2023-06-10T01:51:00.716+0000] {taskinstance.py:1851} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 175, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 193, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/data_ingestion_gcs_multiFile_v02.py", line 33, in format_to_parquet
    table = pv.read_csv(src_file)
  File "pyarrow/_csv.pyx", line 886, in pyarrow._csv.read_csv
  File "pyarrow/_csv.pyx", line 895, in pyarrow._csv.read_csv
  File "pyarrow/error.pxi", line 143, in pyarrow.lib.pyarrow_internal_check_status
  File "pyarrow/error.pxi", line 114, in pyarrow.lib.check_status
OSError: zlib inflate failed: incorrect header check
[2023-06-10T01:51:00.728+0000] {taskinstance.py:1406} INFO - Marking task as UP_FOR_RETRY. dag_id=fhv_taxi_data, task_id=format_to_parquet_task, execution_date=20200802T070000, start_date=20230610T015100, end_date=20230610T015100
[2023-06-10T01:51:00.736+0000] {standard_task_runner.py:107} ERROR - Failed to execute job 145 for task format_to_parquet_task (zlib inflate failed: incorrect header check; 2925)
[2023-06-10T01:51:00.777+0000] {local_task_job.py:164} INFO - Task exited with return code 1
[2023-06-10T01:51:00.808+0000] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-06-11T01:35:40.073+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: fhv_taxi_data.format_to_parquet_task scheduled__2020-08-02T07:00:00+00:00 [queued]>
[2023-06-11T01:35:40.079+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: fhv_taxi_data.format_to_parquet_task scheduled__2020-08-02T07:00:00+00:00 [queued]>
[2023-06-11T01:35:40.079+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2023-06-11T01:35:40.080+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 2
[2023-06-11T01:35:40.080+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2023-06-11T01:35:40.088+0000] {taskinstance.py:1383} INFO - Executing <Task(PythonOperator): format_to_parquet_task> on 2020-08-02 07:00:00+00:00
[2023-06-11T01:35:40.092+0000] {standard_task_runner.py:54} INFO - Started process 7113 to run task
[2023-06-11T01:35:40.096+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'fhv_taxi_data', 'format_to_parquet_task', 'scheduled__2020-08-02T07:00:00+00:00', '--job-id', '716', '--raw', '--subdir', 'DAGS_FOLDER/data_ingestion_gcs_multiFile_v02.py', '--cfg-path', '/tmp/tmpnnyihjzz']
[2023-06-11T01:35:40.097+0000] {standard_task_runner.py:83} INFO - Job 716: Subtask format_to_parquet_task
[2023-06-11T01:35:40.098+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/***/dags/data_ingestion_gcs_multiFile_v02.py
[2023-06-11T01:35:40.159+0000] {data_ingestion_gcs_multiFile_v02.py:20} INFO - dtc_data_lake_dtc-dezc-001 is the bucket.
[2023-06-11T01:35:40.173+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/configuration.py:545: DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
  option = self._get_environment_variables(deprecated_key, deprecated_section, key, section)

[2023-06-11T01:35:40.212+0000] {task_command.py:384} INFO - Running <TaskInstance: fhv_taxi_data.format_to_parquet_task scheduled__2020-08-02T07:00:00+00:00 [running]> on host f41d61956f54
[2023-06-11T01:35:40.233+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:204: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2023-06-11T01:35:40.259+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=admin
AIRFLOW_CTX_DAG_ID=fhv_taxi_data
AIRFLOW_CTX_TASK_ID=format_to_parquet_task
AIRFLOW_CTX_EXECUTION_DATE=2020-08-02T07:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2020-08-02T07:00:00+00:00
[2023-06-11T01:35:40.683+0000] {python.py:177} INFO - Done. Returned value was: None
[2023-06-11T01:35:40.693+0000] {taskinstance.py:1406} INFO - Marking task as SUCCESS. dag_id=fhv_taxi_data, task_id=format_to_parquet_task, execution_date=20200802T070000, start_date=20230611T013540, end_date=20230611T013540
[2023-06-11T01:35:40.791+0000] {local_task_job.py:164} INFO - Task exited with return code 0
[2023-06-11T01:35:40.811+0000] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
